Examples/UIExplorer/NavigationExperimental/NavigationCompositionExample.js

```diff
@@ -120,8 +122,8 @@ const ExampleAppReducer = NavigationReducer.TabsReducer({
   tabReducers: ExampleTabs.map(tab => tab.reducer),
 });
 
-function stateTypeTitleMap(pageState) {
-  switch (pageState.type) {
+function routeTypeTitleMap(route) {
+  switch (route.type) {
```

Comment:
please revert this. It should be called pageState (having the 'route' term around was confusing)